modules/metastorage/README.md

```diff
@@ -0,0 +1,46 @@
+# Metastorage
+
+The module for store and access to metadata. There is module is connected with this one:
```

Comment:
The module for storing and accessing metadata. This module is linked to another one: